sanic/blueprint_group.py

```diff
@@ -0,0 +1,93 @@
+class BlueprintGroup(object):
```

Comment:
it's default to new-style classes in python3, no need `object` here.

Reply:
Done